doc/user/content/sql/alter-cluster.md

```diff
@@ -47,6 +61,22 @@ See the reference documentation for [`CREATE CLUSTER`](../create-cluster/#schedu
 or [`CREATE MATERIALIZED VIEW`](../create-materialized-view/#refresh-strategies)
 for more details on scheduled clusters.
 
+### Reconfiguring Without Downtime
```

Comment:
```suggestion
### No-downtime reconfiguration
```

Reply:
I quite like "graceful reconfiguration".

Reply:
I was also thinking `Zero downtime resize`
I was avoiding graceful reconfig as it was more of an internal name,  but if We're cool with it I also like it.